documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue

```diff
@@ -243,10 +243,7 @@ export default {
       if (this.file && this.file.folder){
         this.$root.$emit('document-open-folder', this.file);
       } else {
-        let id = this.file.id;
-        if (this.file.sourceID){
-          id = this.file.sourceID;
-        }
+        let id = this.file.sourceID ? this.file.sourceID :this.file.id;
```

Comment:
@ahamdi  the `sourceId` is the id of original file so the condition to check if the file is a symlink is already exist ( if sourceId not null so the file is a symlink ) and get the original attachment by the id or the sourceId  that's why the open preview document works correctly , i just clean the code .

Reply:
I agree, and think this is how it was implemented. But the behavior is not correct, we need to be able to preview a symlink the same way we preview original document.